src/process/qgsprocess.cpp

```diff
@@ -545,6 +561,7 @@ void QgsProcessingExec::showUsage( const QString &appName )
       << "\t--version or -v\t\tOutput all versions related to QGIS Process\n"
       << "\t--json\t\tOutput results as JSON objects\n"
       << "\t--verbose\tOutput verbose logs\n"
+      << "\t[-l, --lang language]\tuse language for naming of new fields\n"
```

Comment:
I think we should generalise this -- it will be used for more than just new field names:

```suggestion
      << "\t[-l, --lang language]\tuse language for localised translated strings\n"
```